dist/index.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.1/css/all.min.css"
+    />
+    <title>To do List</title>
+  <script defer src="main.js"></script></head>
+  <body>

```

Comment:
It would be good if you kindly use the `main` tag in your HTML which will be good for your accessibility.

Reply:
Linter checks are very important for removing errors please kinldy also consider linter checks as they are not passing on your repo.